src/backend/renderer/multigpu/mod.rs

```diff
@@ -637,6 +640,10 @@ pub trait GraphicsApi {
     ///
     /// Existing devices are guranteed to be not recreated
     fn enumerate(&self, list: &mut Vec<Self::Device>) -> Result<(), Self::Error>;
+    /// Method to force a re-enumeration, e.g. to free resources
+    fn wants_enumeration(&self) -> bool {
```

Comment:
If this has to force a re-enumeration maybe we should call the function just `force_enumeration`? Or alternative `needs_enumeration` which is a bit less strong.

Reply:
`force_enumeration` sounds like it does what `enumerate` does, just with skipping some checks. `needs_enumeration` sounds better to me.